File: Week 3/W3_T4/Person.java
public class Person {
    public String first_name;
    public String last_name;
    private int age;

    public Person(String fname, String lname, int age) {
        this.first_name = fname;
        this.last_name = lname;
        this.age = age;
    }

    public int getAge() {
        return this.age;
    }

    public void ageUp() {
        this.age += 1;
    }

    public String getFullname() {
        return this.first_name + " " + this.last_name;
    }

    public void printFullname() {
        System.out.println(this.first_name + " " + this.last_name);
    }
}

File: Week 3/W3_T6/person.dart
class Person {
  String first_name;
  String last_name;
  int _age;

  Person(String fname, String lname, int age)
      : first_name = fname,
        last_name = lname,
        _age = age;

  int getAge() {
    return _age;
  }

  void ageUp() {
    _age += 1;
  }

  String getFullname() {
    return first_name + " " + last_name;
  }

  void printFullname() {
    print(first_name + " " + last_name);
  }
}